### ISSUE TITLE
[Bug]: Tablist Accessibility bug, aria-required-children

### ISSUE THREAD

**Issue by Yash5044** [microsoft/fluentui#28351]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
@fluentui/react-components": "9.2.0"
Browser: Microsoft Edge
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://react.fluentui.dev/?path=/docs/components-tablist--default

## Actual Behavior
Running Accessibility Insights web test FastPass on Tablist gives 'aria-required-children' failure. Screenshot attached:
![image](https://github.com/microsoft/fluentui/assets/46025582/f44bba89-676d-43e1-830c-846f3f53ff48)
![image](https://github.com/microsoft/fluentui/assets/46025582/fdb6592e-99fc-4ac5-8a2b-48c03ef5b28a)

## Expected Behavior
There shouldn't be any failure when running FastPass.

## Repro Steps:
1. Go to https://react.fluentui.dev/?path=/docs/components-tablist--default to find the tablist in fluent v9. 
2. Run FastPass web test by Accessibility Insights to see the failure mentioned above.

### Logs

```shell
Title: WCAG 1.3.1: Ensures elements with an ARIA role that require child roles contain them (.___8n9zdb0.f1063pyq[aria-orientation="horizontal"])
Tags: Accessibility, WCAG 1.3.1, aria-required-children

Issue: Ensures elements with an ARIA role that require child roles contain them (aria-required-children - https://accessibilityinsights.io/info-examples/web/aria-required-children)

Target application: Components / TabList - Default ⋅ Storybook - https://react.fluentui.dev/?path=/docs/components-tablist--default

Element path: #storybook-preview-iframe;.fui-FluentProvider239 > div > .___1o1f0dt.f6jr5hl.f1i9oj3a > .___8n9zdb0.f1063pyq[aria-orientation="horizontal"]

Snippet: <div role="tablist" aria-orientation="horizontal" data-tabster="{&quot;mover&quot;:{&quot;cyclic&quot;:true,&quot;direction&quot;:2,&quot;memorizeCurrent&quot;:true}}" class="fui-TabList ___8n9zdb0 f22iagw f1063pyq fi64zpg flvyvdh f10pi13n f1q9h2pe">

Related paths: 
  #storybook-preview-iframe;.fui-FluentProvider239 > div > .___1o1f0dt.f6jr5hl.f1i9oj3a > .___8n9zdb0.f1063pyq[aria-orientation="horizontal"] > i:nth-child(1)
  #storybook-preview-iframe;.fui-FluentProvider239 > div > .___1o1f0dt.f6jr5hl.f1i9oj3a > .___8n9zdb0.f1063pyq[aria-orientation="horizontal"] > i:nth-child(6)

How to fix: 
Fix any of the following:
  Element has children which are not allowed (see related nodes)
  Element has no aria-busy="true" attribute

Environment: Microsoft Edge version 114.0.1823.58

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
```

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by behowell** [comment]
This is a false positive by Accessibility Insights. The items under Related Paths that it's complaining about all have `aria-hidden="true"`. They are not in the accessibility tree and should not be triggering this error.

The issue in axe-core is logged here: https://github.com/dequelabs/axe-core/issues/3850

It is fixed in axe-core v4.7.1, but Accessibility Insights is still using v4.6.3.
